lib/styles/settings/_shadows.scss

```diff
@@ -1,4 +1,13 @@
-@import 'colors';
+@import 'colors/raw';
+
+$color-shadow: rgba(var(--raw-black-rgb), 0.3);
+$color-success-shadow: rgba(var(--raw-green-500-rgb), 0.7);
+$color-danger-shadow: rgba(var(--raw-red-500-rgb), 0.7);
+$color-warning-shadow: rgba(var(--raw-gold-500-rgb), 0.7);
+
+$color-neutrals-shadow-weak: rgba(var(--raw-black-rgb, 0.06));
+$color-neutrals-shadow: rgba(var(--raw-black-rgb, 0.08));
+$color-neutrals-shadow-strong: rgba(var(--raw-black-rgb, 0.12));
 
 $shadow-s: 0 1px 2px $color-neutrals-shadow-weak, 0 8px 16px $color-neutrals-shadow-weak,
```

Comment:
these shadows still use old colors

Reply:
`$color-neutrals-shadow-weak` is defined a few lines above, it uses new colors.